src/test/java/com/google/api/codegen/protoannotations/testdata/java_library.baseline

```diff
@@ -7240,786 +7161,820 @@ public class LibraryServiceStubSettings extends StubSettings<LibraryServiceStubS
     }
   }
 }
-============== file: src/main/java/samples/com/google/gcloud/pubsub/v1/discussbookcallable/DiscussBookCallableCallableStreamingBidiProg.java ==============
-//// [ This is an auto-generated sample file produced by the gapic-generator. Sample name: "DiscussBookCallableCallableStreamingBidiProg" ]
-//// STUB standalone sample "DiscussBookCallableCallableStreamingBidiProg" /////
-
-// FIXME: Insert here set-up comments that we never want to display in cloudsite. These are seen by users perusing the samples directly in the repository.
+============== file: src/main/java/samples/com/google/example/library/v1/discussbookcallable/DiscussBookCallableCallableStreamingBidiProg.java ==============
```

Comment:
I think we are not yet able to generate samples using protoannotations. Is the baseline copied here for testing generating libraries using protoannotations? If so the samples shouldn't be here.

Reply:
This baseline would be generated from using both the GAPIC config and the annotated proto file, so the samples would exist.

This test is to make sure there aren't regressions once we introduce annotations in the proto files (even with the GAPIC config).

Reply:
I'm confused, because the library_gapic.yaml is deleted below.....

Reply:
Yes, there was a library_gapic.yaml in a `protoannotations` test folder, and and another file of the same name in the `gapic` test resource folder. This PR removes the former, so there is only one `library_gapic.yaml` for all tests.

Reply:
Ah, different directory. Gotcha.